Clarifications about role vs user type access (#9858)
newrelic/docs-website: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts.mdx

import userManagementDiagram from 'images/user-management-diagram.png'

> Here we'll explain how New Relic users get access to specific capabilities and specific accounts. 

Reviewer: ```suggestion
Here we'll explain how to give New Relic users access to specific capabilities and accounts.
```

## How user access works: user type and roles [#user-type-groups-relation]

> When it comes to what New Relic features your users can access, there are two main factors: 

Reviewer: ```suggestion
When it comes to what New Relic capabilities your users can access, there are two main factors: 
```

> * A user's **[user type](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type)**: For organizations on [our usage-based pricing](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing), your users' user type is a billing factor. The user type is what sets the maximum allowed capabilities a user can access. It overrides all role-related access. It's meant to be a fairly long-term setting based on someone's expected New Relic duties.

Reviewer: ```suggestion
* **[User type](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type)**: For organizations on [our usage-based pricing](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing), user type is a billing factor. The user type is what sets the maximum allowed capabilities a user can access. It overrides all role-related access. It's meant to be a fairly long-term setting based on someone's expected New Relic duties.
```

> * A user's assigned **roles**: after a user's user type is decided, **roles** can be used to more precisely control a user's access. Roles are sets of **capabilities**, which are the granular New Relic abilities (for example, the ability to modify New Relic APM settings). Roles are assigned by applying them to a [user group](#groups).

Reviewer: ```suggestion
* **Role**: after a user's user type is decided, **roles** can be used to more precisely control a user's access. Roles are sets of **capabilities**. Roles are assigned by applying capabilities to a [user group](#groups).
```

> Let's say a basic user is assigned a group with a role with wide New Relic access, like [**All product admin**](#standard-roles). That user's user type (basic user) takes precedence, and would prevent them from using some of the features that a core user or full platform user would have access to. 

Reviewer: ```suggestion
Let's say a basic user is assigned a group with a role with wide New Relic access, such as the [**All product admin**](#standard-roles) role. That user's user type (basic user) takes precedence, which means that even though they're assigned to an admin-level group, they cannot use features that a a core user or full platform user would have access to. 
```

This doc is focused on role-related permissions. For details about user types, see [User type](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type).

## Groups give users access to roles and accounts [#understand-concepts]

> In order for a New Relic user to be able to access New Relic features, they must be in a [group](#groups), and that group must have access to: 

Reviewer: ```suggestion
In order for a New Relic user to use New Relic features, they must be in a [group](#groups) with assigned access to: 
```
